src/Vulkan/VkCommandList.cpp

```diff
@@ -181,7 +185,7 @@ void VkCommandList::SetLayoutResources(const RHIResourceLayout& layout,
 
     commandBuffer->pushConstants(*vkResourceLayout.pipelineLayout,
                                  stageFlags,
-                                 0,
+                                 layout.GetLocalConstantsOffset(), // Bindless indices start after local constants
```

Comment:
We agree that this will be the opposite after the resourceLayout changes, right?